Directory.Build.targets

```diff
@@ -142,6 +142,14 @@
   <Import Condition="'$(IncludeVSTestReferences)' == 'true' AND ('$(BuildingNETCoreAppVertical)' == 'true' OR '$(BuildingNETFxVertical)' == 'true') AND '$(Language)' != 'VB'" Project="$(PackagesDir)\microsoft.net.test.sdk\$(MicrosoftDotNetTestSdkVersion)\build\netcoreapp1.0\Microsoft.Net.Test.Sdk.targets" />
   <Import Condition="'$(IncludeVSTestReferences)' == 'true' AND ('$(BuildingNETCoreAppVertical)' == 'true' OR '$(BuildingNETFxVertical)' == 'true')" Project="$(PackagesDir)\microsoft.codecoverage\$(MicrosoftDotNetTestSdkVersion)\build\netstandard1.0\Microsoft.CodeCoverage.targets" />
 
+  <ItemGroup>
+    <!-- Should be added to BuildTools but for now adding here to enable netcoreapp3.0 support -->
```

Comment:
NIT: Can we add an issue number to the comment so that the work gets tracked?

Reply:
Better I submitted the PR. https://github.com/dotnet/buildtools/pull/2151